.github/workflows/push-rdev.yml

```diff
@@ -47,4 +47,4 @@ jobs:
           DOCKER_TAG=${DOCKER_TAG#refs/tags/}
           # replace `/` with `_`
           DOCKER_TAG=${DOCKER_TAG//\//-}
-          scripts/happy --profile="" push --extra-tag ${DOCKER_TAG} --extra-tag sha-${GITHUB_SHA:0:8} --extra-tag build-${GITHUB_RUN_NUMBER} ${{ matrix.component }}
+          scripts/happy --profile="" push --stack-name ${DOCKER_TAG} --extra-tag sha-${GITHUB_SHA:0:8} --extra-tag build-${GITHUB_RUN_NUMBER} ${{ matrix.component }}
```

Comment:
What does DOCKER_TAG look like?

Reply:
It is matching branch or tag name that triggered this GHA job. See the `*rdev*` patterns at the top of the file.  By this line it will be something like `rdev-my-branch`.  This ends up being both the rdev stack name and the tag name used by both Portal and Explorer docker images.